lib/ansible/modules/cloud/vmware/vmware_guest_cross_vc_clone.py

```diff
@@ -0,0 +1,292 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2020, Anusha Hegde <anushah@vmware.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+module: vmware_guest_cross_vc_clone
+
+short_description: Cross-vCenter VM/template clone
+
+version_added: '2.10'
+
+description:
+  - 'This module can be used to for Cross-vCenter vm/template clone'
+
+options:
+  name:
+    description:
+      - Name of the virtual machine or template.
+      - This is a required parameter, if parameter C(uuid) or C(moid) is not supplied.
+    type: str
+  uuid:
+    description:
+      - UUID of the instance to clone from, this is VMware's unique identifier.
+      - This is a required parameter, if parameter C(name) or C(moid) is not supplied.
+    type: str
+  moid:
+    description:
+      - Managed Object ID of the instance to manage if known, this is a unique identifier only within a single vCenter instance.
+      - This is required if C(name) or C(uuid) is not supplied.
+    type: str
+  use_instance_uuid:
+    description:
+      - Whether to use the VMware instance UUID rather than the BIOS UUID.
+    default: no
+    type: bool
+  destination_vm_name:
+    description:
+      - The name of the cloned VM.
+      - This is a required parameter.
+    type: str
+  destination_vcenter:
+    description:
+      - The hostname or IP address of the destination VCenter.
+      - This is a required parameter.
+    type: str
+  destination_vcenter_username:
+    description:
+      - The username of the destination VCenter.
+      - This is a required parameter.
+    type: str
+  destination_vcenter_password:
+    description:
+      - The password of the destination VCenter.
+      - This is a required parameter.
+    type: str
+  destination_host:
+    description:
+      - The name of the destination host.
+      - This is a required parameter.
+    type: str
+  destination_datastore:
+    description:
+      - The name of the destination datastore.
+      - This is a required parameter.
+    type: str
+  destination_vm_folder:
+    description:
+      - The name of the destination folder where the VM will be cloned.
```

Comment:
Add an example for this parameter so that the user will know how to use this parameter.